jx_sqlite/__init__.py

```diff
@@ -273,3 +273,11 @@ def copy_cols(cols, nest_to_alias):
         {"gte": [{"length": "nested_path"}, 1]}
     ]}
 )
+
+
+json_types = {
+    "TEXT": "string",
+    "INTEGER": "integer",
+    "REAL": "double",
+    "TINYINT": "boolean"
+}
```

Comment:
I don't see this elsewhere, but there are few similar.
May be we can replace it's use to something else. like:
`ctype={"TEXT": "string", "REAL": "number", "INTEGER": "integer"}.get(dtype)
`